src/main/resources/db/changelog/changes/create-tables.sql

```diff
@@ -0,0 +1,25 @@
+--liquibase formatted sql
+--changeset <romanovosad>:<create-tables>
+
+CREATE TABLE IF NOT EXISTS cameras
+(
+    id         bigserial,
+    name       varchar(255) NOT NULL,
+    nasa_id    bigint       NOT NULL,
+    created_at timestamp    NOT NULL default now(),
+    constraint PK_cameras PRIMARY KEY (id),
+    constraint UQ_cameras_nasa_id UNIQUE (nasa_id)
+);
+
+CREATE TABLE IF NOT EXISTS pictures
+(
+    id         bigserial,
+    nasa_id    bigint       NOT NULL,
+    img_src    varchar(255) NOT NULL,
+    created_at timestamp    NOT NULL DEFAULT now(),
```

Comment:
I consider we should save the "earth_date" value from NASA API, not the date it was stolen :)

Reply:
It is not exactly mentioned, as I understand it is part of table validation.